optional/unbound/unbound.conf

```diff
@@ -8,6 +8,7 @@ server:
   do-udp: yes
   do-tcp: yes
   do-daemonize: no
+  do-not-query-localhost: no
   access-control: {{ SUBNET }} allow
 {%- if SUBNET6 %}
   access-control: {{ SUBNET6 }} allow
```

Comment:
The `access-control SUBNET6` (no []) and the `interface: ::0` above do not work (no newline)
Can you rebase this - I think this was fixed in the meantime.

Reply:
Nothing was changed here... so if it's broken it was broken elsewhere.

Reply:
Ah, I've marked the wrong line, I was thinking about `{{- '  interface: ::0' if SUBNET6 }}`.
I'll fix it. It creates an unbound.conf like this:
```
server:
  verbosity: 1
  interface: 0.0.0.0  interface: ::0
  logfile: ""
  do-ip4: yes
```